samples/subsys/mgmt/mcumgr/smp_svr/sample.yaml

```diff
@@ -94,3 +94,16 @@ tests:
       - mg100
     integration_platforms:
       - nrf52840dk_nrf52840
+  sample.mcumgr.smp_svr.bt.nrf54l15pdk.ext_flash:
+    extra_args:
+      - OVERLAY_CONFIG="overlay-bt.conf"
+      - DTC_OVERLAY_FILE="boards/nrf54l15pdk_ext_flash.overlay"
+      - mcuboot_CONF_FILE="boards/nrf54l15pdk_ext_flash.conf"
+      - mcuboot_EXTRA_DTC_OVERLAY_FILE="boards/nrf54l15pdk_ext_flash.overlay"
```

Comment:
Does this work? According to the comment on the main post, this will not work

Reply:
I had added overlay to the sdk-mcuboot repo.